src/internet/subsonic/subsonicservice.cpp

```diff
@@ -494,6 +495,15 @@ void SubsonicLibraryScanner::OnGetAlbumFinished(QNetworkReply* reply) {
   }
 }
 
+void SubsonicLibraryScanner::OnGetAlbumCoverFinished(QNetworkReply* reply, QString albumid) {
+  reply->deleteLater();
+  pending_requests_.remove(reply);
+
+  QByteArray image = reply->readAll();
+
+  service_->emitImageLoaded(albumid, QImage::fromData(image));
```

Comment:
Don't do this. Instead add a signal to `SubsonicLibraryScanner` and connect that signal directly to the `ImageLoaded` signal in `SubsonicService`.